cypress/e2e/settings.spec.ts

```diff
@@ -124,6 +133,40 @@ describe("Settings page", () => {
     cy.get("#title").should("have.value", TEST_TITLE)
   })
 
+  it("should be able to update the SEO to a valid value", () => {
+    // Arrange
+    const expected = "www.space.open.gov.sg"
+    cy.contains("label", "SEO").parent().parent().find("input").as("seoInput")
+
+    // Act
+    cy.get("@seoInput").clear().type(expected)
+
+    // Assert
+    cy.saveSettings()
+    cy.get("@seoInput").should("have.value", expected)
+  })
+
+  it("should not be able to input the protocol at the beginning of the url", () => {
```

Comment:
I think we can add at least 2 more invalid inputs:
1. A domain with a subdirectory (like `open.gov.sg/space/`)
2. An invalid domain (some random characters)

Reply:
Hmm, what properties does the regex ensure? I think we should write invalid test cases given the invariants and not the other way round

Reply:
The regex mainly ensures that the user provides a proper domain name, so it will prevent cases like specifying a subdirectory. I think maybe just having at least this invalid input should be sufficient, so that we don't give users the idea that it is possible to host an Isomer site within a subdirectory?

Reply:
fair enough, added extra test cases!